typescript/infra/scripts/middleware/deploy-token-bridge.ts

```diff
@@ -0,0 +1,48 @@
+import path from 'path';
+
+import {
+  HyperlaneCore,
+  TokenBridgeDeployer,
+  objMap,
+  tokenBridgeFactories,
+} from '@hyperlane-xyz/sdk';
+
+import { circleBridgeAdapterConfig } from '../../config/environments/testnet2/token-bridge';
+import { deployWithArtifacts } from '../../src/deploy';
+import { getConfiguration } from '../helloworld/utils';
+import {
+  getCoreEnvironmentConfig,
+  getEnvironment,
+  getEnvironmentDirectory,
+} from '../utils';
+
+async function main() {
+  const environment = await getEnvironment();
+  const coreConfig = getCoreEnvironmentConfig(environment);
+  const multiProvider = await coreConfig.getMultiProvider();
+  const core = HyperlaneCore.fromEnvironment(environment, multiProvider as any);
+
+  const dir = path.join(
+    getEnvironmentDirectory(environment),
+    'middleware/token-bridge',
+  );
+
+  // config gcp deployer key as owner
+  const ownerConfigMap = await getConfiguration(environment, multiProvider);
+
+  const deployer = new TokenBridgeDeployer(
+    multiProvider,
+    objMap(circleBridgeAdapterConfig, (chain, conf) => ({
+      bridgeAdapterConfigs: [conf],
+      ...ownerConfigMap[chain],
+    })),
+    core,
+    'asdssds',
+  );
+
+  await deployWithArtifacts(dir, tokenBridgeFactories, deployer);
```

Comment:
Should we add these addresses to the SDK now like we did for other middleware?

Reply:
I feel like in an ideal world we would just write them directly, but for now, I just do the `mergeWithSdkContractAddressArtifacts` function right?

Reply:
Yes and you'll also want to expose it in the `hyperlaneCoreAddresses` map:
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/1210/files#diff-8a8d0a4753bc69e9ad052d746b5d3704618d541a46034d2e0449bdb5f651958dR20

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/1215/commits/a08c14ffa7034077b56d7c497fbeadf049d28ab0